meeting-notes/20181113.md

```diff
@@ -0,0 +1,203 @@
+     - pinner should only get state: https://github.com/ipfs-shipyard/peer-star-app/pull/106
+     - test pinner: https://github.com/ipfs-shipyard/peer-star-app/pull/108
+     - membership heuristic sampling function should be synchronous: https://github.com/ipfs-shipyard/peer-star-app/pull/110
+     - chore: using aegir lint and fixes linting errors: https://github.com/ipfs-shipyard/peer-star-app/pull/116
+ - In progress:
+    - expose collaboration replication events: https://github.com/ipfs-shipyard/peer-star-app/pull/111
+ - Blocked:
+ - Next:
+    - continue on exposing collaboration replication events: https://github.com/ipfs-shipyard/peer-star-app/pull/111
+    - support different strategies for local store: https://github.com/ipfs-shipyard/peer-star-app/issues/117
+
+#### @satazor
+ - Concluded:
+   - working group 2019 roadmap, 1st iteration
+   - discussify:
+     - UI level QA: https://github.com/ipfs-shipyard/pm-discussify/issues/58
+     - Standardized styleguide comment names with @andreforsousa
+ - In progress:
+   - Replies
+   - Lazy load of comments and replies (view more)
+ - Next:
+   - Implement "show history" in the UI
+   - Add persistance of the data (pinner + sync UI language)
+     
+Note: Ask pedro about granularity of Discussify v1 milestones on the roadmap
+     
+ #### @andreforsousa    
+ - Concluded:
+    - Discussify:
+    - Style-guide updates with all scenarios and states (hover, focus, error feedback) for "comment-component"
+        - Comment
+        - Comment (ownership)
+        - Comment Actions
+        - Comment Edited
+        - Comment Edited History
+        - Comment Placeholder
+        - Comment Removed
+        - Comment Input (new reply)
+        - Comment Input (editing)
+    - QA & style-guide updates for the pre-alpha version
+        - FAB Shadow fix
+    - PeerPad UX/UI fixes:
+        - Grid width for 80columns accommodation (desktop)
+        - Dismissible notifications (desktop)
+        - Empty state illustration
+        - History revision
+        
+- Blocked:
+    - PeerPad mobile:
+    - Hi-Fi prototype to start looking for:
+        - Transitions between elements
+        - Start defining values for the eases
+        - This will be in the backlog until desktop version is properly defined and polished, following David’s feedback.
+    
+- In progress:
+  - Discussify Brand & Web Guidelines (manual)
+  - Look at new features for PeerPad (sync with Victor)
+
+ 
+#### @VictorBjelkholm
+ - Concluded:
+ 		- Error reporting on peerpad.net
+ - In Progress:
+ 		- UX review
+    - Pinner review
+ - Blocked:
+ - Next:
+    - Pinner deployment
+    - Metrics
+ 
+#### @aschmahmann
+ - Concluded:
+  - Roadmapping round #1
+  - Basic graph sync + updating working
+ - In progress:
+  - Looking into using existing CRDT tools like y.js to talk to the local Go graph syncing client
+     - Any advice would be appreciated
```

Comment:
@pgte has some very good ideas for this as he has done it multiple times for other projects.

In summary, you should be able to sync any graph super quickly just by opening a libp2p connection and speaking a custom protocol. This is the shortcut until GraphSync is done.

Reply:
This is essentially what I'm doing for now. As you mentioned above, figuring out how much overlap the synchronization of version graphs that I am working on has with GraphSync is still something we need to work out. I sat in on the ipld biweekly last week to get some info on how GraphSync is progressing.